src/utils/api.js

```diff
@@ -1,8 +1,18 @@
-const apiCall = async (authContext, endpoint) => {
+const apiCall = async (authContext, endpoint, body) => {
     const user = await authContext.getCurrentUser();
     const jwtToken = user.signInUserSession.idToken.jwtToken;
-    const res = await fetch(`/${endpoint}?id_token=${jwtToken}`);
+    var req;
 
+    if (body !== undefined) {
+        const bodyWithToken = JSON.stringify(Object.assign({}, body, {id_token: jwtToken}));
+        const headers =  {'Content-Type': 'application/json'}
+        req = new Request(`/${endpoint}`, {method: 'POST', body: bodyWithToken, headers: headers});
+    } else {
+        req = `/${endpoint}?id_token=${jwtToken}`;
```

Comment:
Arent these two req vars different types? The first a Request object, the second a string. Does the fetch method handle both types?

Reply:
Yes it does.

Reply:
I wanted to go for "least change" so maintained the string for most use cases.

Reply:
Ah ok, so the first req is a POST and the 2nd is a GET. Maybe a comment above each to clarify would be grand.